### PR TITLE
Bazel 6.0 blog

### PR DESCRIPTION
**Pull request by radvani13**
Blog post for Bazel 6.0 

https://docs.google.com/document/d/1d-02eFtMpqJziRTDSF-W_WqwSAvrlqyHTlMuCWd2vBU/edit?resourcekey=0-_d-GsecTFvFeQgCoomYxVw#heading=h.hynlhcx42ii6

### CODE REVIEW

File: _posts/12-06-2022-Bazel-release-6.0-blog.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+-   Support for `rules_jvm_external`, allowing users to download Maven dependencies for Java projects.
+  
+## Build Android apps with Bazel
+By popular customer request, Bazel 6.0 uses [D8](https://developer.android.com/studio/command-line/d8) for dexing and desugaring by default. With D8, you leverage the latest Android tooling when building mobile apps. Through [community contributions,](https://github.com/bazelbuild/bazel/pulls?q=is%3Apr+is%3Aclosed+label%3Ateam-android+closed%3A2021-11-01..2022-12-02+) Bazel Android builds now also support a variety of quality-of-life and efficiency improvements, such as persistent workers for resource processing and (optional) manifest permission merging.
+
+## Optional toolchains
+Our Developer Satisfaction survey indicated that rule authors want support for improved [toolchain development](https://bazel.build/versions/6.0.0/extending/toolchains#optional-toolchains). Bazel 6.0 allows authors to write rules using an [optional, high performance toolchains](https://bazel.build/docs/toolchains#optional-toolchains)s when available, with a fallback implementation for other platforms.
```
Comment (jin) [inline _posts/12-06-2022-Bazel-release-6.0-blog.md:25]: remove extra "s":

```suggestion
Our Developer Satisfaction survey indicated that rule authors want support for improved [toolchain development](https://bazel.build/versions/6.0.0/extending/toolchains#optional-toolchains). Bazel 6.0 allows authors to write rules using an [optional, high performance toolchains](https://bazel.build/docs/toolchains#optional-toolchains) when available, with a fallback implementation for other platforms.
```

File: _posts/12-06-2022-Bazel-release-6.0-blog.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+## New build performance profiles
+[Optimize for build productivity](https://blog.bazel.build/2022/11/15/build-performance-metrics.html) with new system and CPU related metrics data.
+
+-   system load average (`--experimental_collect_load_average_in_profiler`)
+-   worker memory usage (`--experimental_collect_worker_data_in_profiler`)
+-   system network usage (`--experimental_collect_system_network_usage`)
+    
+
+## Control `.bzl` dependencies with load visibility
+Now, rule and macro authors can declare a [load visibility](https://bazel.build/versions/6.0.0/concepts/visibility#load-visibility) for their `.bzl` files by calling the new `[visibility()](https://bazel.build/versions/6.0.0/rules/lib/globals#visibility)` built-in function. This restricts what parts of the workspace may `load()` the `.bzl` file, so that macros and rules do not automatically become public APIs.
```
Comment (jin) [inline _posts/12-06-2022-Bazel-release-6.0-blog.md:36]: Bad formatting for the visibility() function link -- the backticks should be in the square brackets.

File: _posts/12-06-2022-Bazel-release-6.0-blog.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+[Release notes for Bazel 6.0](https://docs.google.com/document/d/1pu2ARPweOCTxPsRR8snoDtkC9R51XWRyBXeiC6Ql5so/edit#heading=h.kh1neevharzp)
```
Comment (jin) [inline _posts/12-06-2022-Bazel-release-6.0-blog.md:38]: nit: "Read the full release notes for Bazel 6.0"

File: _posts/12-06-2022-Bazel-release-6.0-blog.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+Today, we launch Bazel 6.0 LTS!. Bazel 6.0 delivers many requested customer updates to simplify Bazel usage and eliminate common Android app development roadblocks.
+
+![Image](../images/Bazel6.0ReleaseBlogPost.png)
+
+### Bzlmod becomes Generally available
+[Bzlmod](https://bazel.build/docs/bzlmod) automatically resolves transitive dependencies, allowing projects to scale while staying fast and resource-efficient. In development since 2021, Bzlmod enters general availability with Bazel 6.0.
+
+-   Pipelines to automatically check common dependencies and rulesets into the Bazel Central Registry.
+-   Updated [Bzlmod Migration Guide](https://docs.google.com/document/d/1JtXIVnXyFZ4bmbiBCr5gsTH4-opZAFf5DMMb-54kES0/edit?usp=gmail) includes scripts and documentation for migration.
+-   Support for `rules_jvm_external`, allowing users to download Maven dependencies for Java projects.
```
Comment (jin) [inline _posts/12-06-2022-Bazel-release-6.0-blog.md:19]: This line reads like rules_jvm_external is new. How about "Added Bzlmod support for..."?

File: _posts/12-06-2022-Bazel-release-6.0-blog.md (hunk 2 above)
Comment (radvani13) [reply _posts/12-06-2022-Bazel-release-6.0-blog.md]: resolved

**Comment by meteorcloudy** [comment]
Can you fix the date in the picture? Also please use an unambiguous date format

**Review by aiuto — COMMENTED** [review]
LGTM, same comment as Yun about the date.

**Review by aiuto — APPROVED** [review]

**Comment by meteorcloudy** [comment]
We have identified a new release blocker for 6.0: https://github.com/bazelbuild/bazel/issues/16933, so please hold a bit from publishing the blog.

**Pull request merged**